### DISCUSSION TITLE
BUILD FAILED

### DISCUSSION THREAD

**Discussion by Abinash6000** [oppia/oppia-android#4858] [Q&A (Installation)] [answered]
Sorry, but I don't how to proceed furthur. Please help I am new.
![image](https://user-images.githubusercontent.com/110354418/214572286-e678d1d7-09f1-48de-9798-47c35e9c6276.png)
![image](https://user-images.githubusercontent.com/110354418/214572295-e7cecee0-9958-4f2f-9f19-4a7ae92a13ac.png)
![image](https://user-images.githubusercontent.com/110354418/214572305-a6d498ea-89fd-4237-9809-308eeb06dac8.png)

**Comment by MohitGupta121** [comment]
@Abinash6000 Which Opearting system and android studio version you using? Also please keep internet connected while first time build.

**Reply by Abinash6000** [reply]
Windows Operating System.
Chipmunk 2021.2.1
I know the prefered version is bumble bee but I can't find the version on google.
And I kept the internet connected while the build.

**Reply by Abinash6000** [reply]
Thank you for replying...

**Reply by MohitGupta121** [reply]
You can download that version from [here](https://developer.android.com/studio/archive)
If you get any error feel free to ask here.
Try this and make sure you are in latest commit.

**Reply by Abinash6000** [reply]
![image](https://user-images.githubusercontent.com/110354418/215093655-cddb8f1f-ec63-4508-a0da-b243cbb14c5a.png)
The error persists.

**Reply by MohitGupta121** [reply]
Can you try to Invalidate cache and restart in Android studio. Or re-clone the project in your local. I think you missing something initially.

**Reply by Abinash6000** [reply, marked as answer]
You were right, I was missing something initially. I changed the graddle-wrapper.properties. This was causing the error.
Thank you for your time Mohit. And sorry too.

**Reply by MohitGupta121** [reply]
@Abinash6000 great, means your issue is now resolved? If it resolved please mark this as answer so that it help to other in the community.

And feel free to ask other query in new discussion.
Thanks.
